test/storage_snapper_configure_snapper_test.rb

```diff
@@ -22,13 +22,7 @@
 
     Yast::Storage.stub(:GetEntryForMountpoint).with("/").once.and_return(data)
 
-    Yast::SCR.stub(:Execute).and_return(1)
-    Yast::SCR.should_receive(:Execute).exactly(2).times.and_return(0)
-
-    Yast::SCR.stub(:Write).and_return(1)
-    Yast::SCR.should_receive(:Write).exactly(2).times.and_return(0)
-
-    Yast::StorageUtils.ConfigureSnapper()
+    expect(Yast::StorageSnapper.ConfigureSnapper?).to eq true
```

Comment:
@aschnell Fair enough (although I was actually not talking about checking the whole exact command). But I still can see some things that are not properly tested/described.

For ```ConfigureSnapper?``` we actually have three posibilities (no btrfs, btrfs with snapshots, btrfs without snapshots). Looking at the output of the current test is far from obvious.

For steps 1 and 4, the (un)loading of the extension if needed.